west.yml

```diff
@@ -78,6 +78,16 @@ manifest:
       groups:
          - hal
 
+    - name: aipl
```

Comment:
Place the library directly to SDK. No need to create a dependency to another repo.

Reply:
And I was wrong in this case. Let' revert back to your original idea.